crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -228,6 +229,24 @@ def cancel
         }
       }, status: cancel_upgrade[:status]
     end
+  rescue Crowbar::Error::UpgradeCancelError => e
```

Comment:
the exceptions from `revert_nodes_from_crowbar_upgrade` will be caught in the `rescue StandardError` block. and what exactly do you think is inconsistent here?